src/gui/qgsmaptool.h

```diff
@@ -133,6 +133,12 @@ class GUI_EXPORT QgsMapTool : public QObject
     //! gesture event for overriding. Default implementation does nothing.
     virtual bool gestureEvent( QGestureEvent *e );
 
+    /**
+     * Tooltip event for overriding. Default implementation does nothing.
+     * \since QGIS 3.22
```

Comment:
"Returns TRUE if the event was handled by the tool and should not be propagated further"